orca-sql/src/main/resources/db/changelog/20200327-deleted-executions-table.yml

```diff
@@ -1,10 +1,8 @@
 databaseChangeLog:
   - changeSet:
+      validCheckSum: 8:10687930e669629d8c370cd5a2348585
       id: create-deleted-executions-table
       author: mvulfson
-      validCheckSum:
```

Comment:
Is the idea that this was never used because it's a list instead of a string?  I'd rather keep this around if possible.

Reply:
We can't have two tags with the same name

Reply:
You mean two `validCheckSum` properties?  Yeah, I figure we can have only one.  My question was whether it can be a list, but from https://docs.liquibase.com/concepts/changelogs/changeset-checksums.html#yaml_example apparently not.  So folks must have migrated enough to not need the one we're removing.